src/screens/ChannelListScreen/hooks/useFollowUser.ts

```diff
@@ -39,6 +39,12 @@ const useFollowUser = () => {
     return Boolean(followContext?.users?.find((user) => user?.user_id_followed === targetUser?.id));
   };
 
+  const isFollowingUser = (user_id_followed: string): boolean => {
+    return Boolean(
+      followContext?.users?.find((user) => user?.user_id_followed === user_id_followed)
+    );
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isFollowingUser` function seems to be a duplicate of the `isFollowing` function. Both functions are doing the same thing: checking if a user is being followed by comparing `user_id_followed` with the provided user id. Consider removing one of them to avoid redundancy and improve maintainability.
```diff
-  const isFollowingUser = (user_id_followed: string): boolean => {
-    return Boolean(
-      followContext?.users?.find((user) => user?.user_id_followed === user_id_followed)
-    );
-  };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->